templates/common/on-prem/files/etc-hosts.yaml

```diff
@@ -0,0 +1,7 @@
+mode: 0644
+path: "/etc/hosts"
+contents:
+  inline: |
+      127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
```

Comment:
I guess we should use $cluster_name and $domain_name instead of 'ostest.test.metalkube.org' , right?

Reply:
Whoops, yeah.